src/hooks/useImageUpload.ts

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import instance from 'apis/httpClient';
+
+function useImageUpload() {
+  const [imageUrl, setImageUrl] = useState<string>('');
```

Comment:
```suggestion
  const [imageUrl, setImageUrl] = useState('');
```

타입 추론을 해주고 있기 때문에 불필요한 제네릭 타입 선언인 것 같아요!

Reply:
imageUrl이 원래 인터페이스를 사용하려고 했어서 <> 있었는데 인터페이스를 사용하지 않을때는 지우는게 맞는것 같습니다. 좋은 지적 감사합니다.